backend/product/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from core.models import MainModel
+
+class Brand(MainModel):
+    name = models.CharField(max_length=150, null=False)
+
+class Truck(MainModel):
```

Comment:
There are a couple more fields that we will need:
- Unicode ID
- Chassis Number
- Model Number
- Description